### PR TITLE
Adding a README.md file

### PR DESCRIPTION
**Pull request by smmnaghibi**
Adding a README.md file
by MuhammadMahdi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# My Public IP
+
+## About The Project
+This program will give you, your **Public IP**
```
Comment (pouriya) [inline README.md:4]: program or library?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Build With
+* Python
```
Comment (pouriya) [inline README.md:7]: We use python **3.x** only

File: README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Getting Started
+Go to the Project Directory
+
+### Prerequisites
+For installing setup.py you need ```setuptools``` python library
+If you don't have this library on your system, try the command below
+
+```pip install setuptools```
```
Comment (pouriya) [inline README.md:16]: on some systems it's `pip3`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+### Installation
+1. While you're in Project Directory, write the message below in the terminal
+
+    ```make install```
+
+   *Note* : You can also use ```sudo make install```
```
Comment (pouriya) [inline README.md:23]: *you **may** need to use...

File: README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* [```my_ip_public.ipify_org()```](https://ipify.org)
+* [```my_ip_public.my_ip_com()```](https://myip.com)
+* [```my_ip_public.my_ip_io()```](https://my-ip.io)
+
+4. You can always clean this, by ```make clean``` command
+
+### Usage
+This is the example of how to use them
+```
+muhammadmahdi:~/MyFiles/git/13/my_ip_public$ python3
```
Comment (pouriya) [inline README.md:38]: please remove your name and set a generic path name

**Comment by pouriya** [comment]
I merge this PR and let you and others to improve it.

File: README.md (hunk 1 above)
Comment (smmnaghibi) [reply README.md:4]: library

File: README.md (hunk 2 above)
Comment (smmnaghibi) [reply README.md:7]: Updated

File: README.md (hunk 3 above)
Comment (smmnaghibi) [reply README.md:16]: Updated

File: README.md (hunk 4 above)
Comment (smmnaghibi) [reply README.md:23]: Updated

File: README.md (hunk 5 above)
Comment (smmnaghibi) [reply README.md:38]: Updated

**Comment by smmnaghibi** [comment]
Changes added to README.md file

**Pull request merged**